packages/teams-js/test/public/media.spec.ts

```diff
@@ -439,6 +440,133 @@ describe('media', () => {
+          expect(message).not.toBeNull();
+          expect(message.args.length).toBe(1);
+
+          const callbackId = message.id;
+          mobilePlatformMock.respondToMessage({
+            data: {
+              id: callbackId,
+              args: [undefined],
+            },
+          } as DOMMessageEvent);
+
+          expect(mediaError).toBeFalsy();
+        });
+      });
+
+      it('videoController notifyEventToHost is not handled successfully', async () => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(async () => {
+          mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
+          const stopRes = new media.VideoController().stop();
+          const err: SdkError = {
+            errorCode: ErrorCode.INTERNAL_ERROR,
+          };
+          const message = mobilePlatformMock.findMessageByFunc('media.controller');
+          expect(message).not.toBeNull();
+          expect(message.args.length).toBe(1);
+
+          const callbackId = message.id;
+          mobilePlatformMock.respondToMessage({
+            data: {
+              id: callbackId,
+              args: [err],
+            },
+          } as DOMMessageEvent);
+
+          expect(stopRes).rejects.toBe(err);
+        });
+      });
+
+      it('should invoke correct video callback for MediaControllerEvent when registered', () => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.ios).then(() => {
+          mobilePlatformMock.setClientSupportedSDKVersion(nonFullScreenVideoModeAPISupportVersion);
+          const mockCallback = jest.fn();
+          const videoControllerCallback: media.VideoControllerCallback = {
+            onRecordingStarted() {
+              mockCallback();
+            },
+          };
+          const videoProps: media.VideoProps = {
+            videoController: new media.VideoController(videoControllerCallback),
+          };
+          const mediaInputs: media.MediaInputs = {
+            mediaType: media.MediaType.Video,
+            maxMediaCount: 10,
+            videoProps: videoProps,
+          };
+
+          media.selectMedia(mediaInputs).then(result => {
+            const message = mobilePlatformMock.findMessageByFunc('selectMedia');
+            expect(message).not.toBeNull();
+            expect(message.args.length).toBe(1);
+
+            const callbackId = message.id;
+            mobilePlatformMock.respondToMessage({
+              data: {
+                id: callbackId,
+                args: [undefined, undefined, 1],
+              },
+            } as DOMMessageEvent);
+
+            expect(result).toBeDefined;
+            expect(mockCallback).toHaveBeenCalled();
+          });
+        });
+      });
+
+      it('should not invoke video callback for MediaControllerEvent when not registered', async () => {
```

Comment:
same as above

Reply:
Included in a bug item, will link to it in teams git magic channel